mlrun/api/api/endpoints/client_spec.py

```diff
@@ -24,7 +26,14 @@
     "/client-spec",
     response_model=mlrun.api.schemas.ClientSpec,
 )
-def get_client_spec():
-
-    # TODO: cache me
```

Comment:
still needed, no?

Reply:
depends, as it now will have a dynamic response based on the client, so this will mean we will need to cache per client combination, also we want to reduce the memory usage of the API, adding stuff like this won't exactly help us with that

Reply:
also note that this endpoint doesn't have access to the database, it mainly queries data from the config data or other global variables we have, so it pretty much very light weight